lib/action_cable/connection/base.rb

```diff
@@ -12,16 +12,16 @@ module Connection
     #   module ApplicationCable
     #     class Connection < ActionCable::Connection::Base
     #       identified_by :current_user
-    #   
-    #       def connect
+    #
+    #       after_connect do
```

Comment:
I'm not sure I agree with turning `connect` and `disconnect` into a callback. As methods they're nice anchors which give feeling of how connection (or disconnection) happens.

If it's all callbacks it loses meaning. I.e. imagine if Active Record's actual saving was just implemented as the last `before_save` or the first `after_save`, sounds like that would be complex down the road.

Though I'm definitely :+1: on adding before and after callbacks, I think we should keep the `connect` and `disconnect` methods.